packages/manager/modules/pci/src/projects/project/project.html

```diff
@@ -96,42 +96,83 @@
+                                                        data-ng-href="{{ ::link.href }}"
+                                                        target="_blank"
+                                                        rel="noopener"
+                                                        data-translate="{{ ::link.description }}"
+                                                    ></a>
+                                                </oui-tile-description>
+                                            </oui-tile-definition>
+                                        </oui-tile>
                                     </div>
                                     <div
-                                        class="col-sm-6 mb-4"
+                                        class="col-sm-4 mb-4"
                                         data-ng-if="$ctrl.coreConfig.getRegion() !== 'US'"
                                     >
-                                        <h2
-                                            data-translate="pci_projects_project_community"
-                                        ></h2>
-                                        <ul>
-                                            <li
+                                        <oui-tile
+                                            heading="{{ 'pci_projects_project_community' | translate}}"
+                                        >
+                                            <oui-tile-definition
                                                 data-ng-repeat="communitylink in $ctrl.communityLinks track by $index"
                                             >
-                                                <a
-                                                    data-ng-href="{{ ::communitylink.href }}"
-                                                    target="_blank"
-                                                    rel="noopener"
-                                                    data-translate="{{ ::communitylink.translation }}"
-                                                ></a>
-                                            </li>
-                                        </ul>
+                                                <oui-tile-term
+                                                    data-translate="{{communitylink.term | translate}}"
+                                                ></oui-tile-term>
+                                                <oui-tile-description>
+                                                    <a
+                                                        data-ng-href="{{ ::communitylink.href }}"
+                                                        target="_blank"
+                                                        rel="noopener"
+                                                        data-translate="{{ ::communitylink.description }}"
+                                                    ></a>
```

Comment:
```suggestion
                                                <oui-tile-description>
                                                    <a
                                                        data-ng-href="{{ ::communitylink.href }}"
                                                        target="_blank"
                                                        rel="noopener"
                                                    >
                                                       <span data-ng-bind=":: communitylink.description"></span>
                                                    </a>
```

Reply:
Thanks for the suggestion but this is just a compact way to write an HTML anchor. 
It's used in the same file in the line 142 and works fine.